doc/hand_eye_calibration/hand_eye_calibration_tutorial.rst

```diff
@@ -0,0 +1,142 @@
+as the target is stationary in the robot's base frame, the hand-eye calibration can be estimated from a sequence of 5 or
+more poses.
+
+In the "Target Params" section of the "Target" tab, we will use the default target parameters:
+
+- **markers, X**: 3
+- **markers, Y**: 4
+- **marker size (px)**: 200
+- **marker separation (px)**: 20
+- **marker border (bits)**: 1
+- **ArUco dictionary**: DICT_5X5_250
+
+Press the "Create Target" button to create the target image:
+
+.. image:: images/aruco_target_handeye_panel.png
+
+Save the target image using the "Save Target" button, and print out the image. Feel free to experiment with the target
+parameters to see how they affect the target, but be sure to remember the parameters used for the target you print--you
+will need to input the same parameters for the target to be recognized.
+
+The target must be flat to be reliably localized by the camera. Laying it on a flat surface is sufficient, or it can be
+mounted to a board. Measure the marker width (the outside dimension of one of the black squares), as well as the
+separation distance between markers. Enter these values, in meters, in the appropriate boxes in the "Target Params"
+section. Also, select the appropriate topics in the "Image Topic" and "CameraInfo Topic" drop-down menus.
+
+Finally, place the target near the robot, where it can be easily seen by the camera.
+
+Geometric Context
+-----------------
+The second tab, labeled "Context", contains the geometric information necessary to conduct the calibration. Set the
+"Sensor configuration" to "Eye-in-hand", then select the appropriate frames. The "Sensor frame" is the camera optical
+frame (using the right-down-forward standard, as specified in `REP 103 <https://www.ros.org/reps/rep-0103.html>`_), the
+"Object frame" is the frame defined by the calibration target, which is called "handeye_target" by default, and the
+remaining frames are the robot links rigidly attached to the camera (the "End-effector frame") and the calibration
+target (the "Robot base frame"; although the target might not be literally "rigidly attached", it doesn't move in this
+frame).
+
+.. image:: images/context_tab.png
+
+The FOV section controls the rendering of the camera's field of view in RViz. To see the FOV, add a "MarkerArray"
+display, and set it to listen to the "/rviz_visual_tools" topic. (It may not appear immediately.)
+
+Finally, it is not necessary to set an initial guess for the camera pose, but it is worth noting that once a calibration has been
+calculated, these fields will be updated with the new calibration.
+
+Collect Dataset
+---------------
+Next, we will capture a calibration dataset. The robot kinematics provide the end-effector's pose in the robot base
+frame, and the calibration target's pose in the camera frame can be estimated, as mentioned above. If the target's pose
+in the robot base frame were known accurately, only a single observation of the camera-target transform would be
```

Comment:
Might be easier to just say "We need to take multiple images to obtain a high-quality calibration result.", although I like the link to the paper

Reply:
Taking multiple images isn't merely to reduce noise and improve the calibration quality, it also makes it possible to get a calibration without knowing the precise target location. I've seen smart people get confused by that, so I'd like to keep this paragraph as is.

Reply:
Is there an issue with writing e.g. `Next, we will capture a calibration dataset. We need to take multiple images to obtain a high-quality calibration result. The robot kinematics provide the end-effector's pose in the robot base...`? I feel like the paragraph dives into the details pretty quickly, and a simple instruction "We will take multiple images" seems clearer than "calibration dataset". I'm thinking of non-native speakers and non-experts.

Reply:
Oh yes, that's a good idea. I added a sentence.